rfc/rfc003-oauth2-authorization.md

```diff
@@ -96,8 +96,8 @@ Example of the actor's DID document:
 
 #### 4.1.2 Server registration
 
-Each service MUST define an `oauth` serviceEndpoint. This endpoint is either the URL of the authorization server, or in case of a compound service an URI which resolves to another service.
-In order for the client to resolve the authorization server endpoint it MUST look up the `auth` service endpoint of the service.
+Each service MUST define an `oauth` serviceEndpoint. This endpoint refers to another service in the DID Document with the type: `oauth`.
```

Comment:
> Each service

"Service" is ambigious: OAuth service ("server") or DID service?

> refers to another service in the DID document

the same DID document, or a parent's?

Reply:
Added notion of compound service (also in terminology)